internal/olm/operator/registry/index/registry_pod.go

```diff
@@ -313,20 +329,29 @@ opm registry add -d {{ $.DBPath }} -b {{ $item.ImageTag }} --mode={{ $item.AddMo
 opm registry serve -d {{ .DBPath }} -p {{ .GRPCPort }}
 `
 
+const fbcCmdTemplate = `mkdir -p {{ .FBCDir }} && \
+echo '{{ .FBCContent }}' >> {{ .FBCFile  }} && \
+opm serve {{ .FBCDir }} -p {{ .GRPCPort }}
+`
+
 // getContainerCmd uses templating to construct the container command
 // and throws error if unable to parse and execute the container command
 func (rp *RegistryPod) getContainerCmd() (string, error) {
-
+	var t *template.Template
 	// create a custom dirname template function
 	funcMap := template.FuncMap{
 		"dirname": path.Dir,
 	}
 
 	// add the custom dirname template function to the
 	// template's FuncMap and parse the cmdTemplate
-	t := template.Must(template.New("cmd").Funcs(funcMap).Parse(cmdTemplate))
+	if rp.HasFBCLabel {
+		t = template.Must(template.New("cmd").Funcs(funcMap).Parse(fbcCmdTemplate))
+	} else {
+		t = template.Must(template.New("cmd").Funcs(funcMap).Parse(cmdTemplate))
```

Comment:
Why not instead of have 2 commands to keep maintained we do not change it for:

```go
if isSQLIndex { 
      // convert to FBC index 
}
```
So that no matter the format we do the same steps and work on in the same.

Reply:
@camilamacedo86,

Based on some of the discussions we had recently with @joelanford, rendering SQLite index images to FBCs in SDK code is not a viable approach due to cross-compilation conflicts with Operator Registry code.

Reply:
@camilamacedo86 

Adding [comment](https://github.com/operator-framework/enhancements/pull/112#discussion_r856412131) from the EP for your reference